### PR TITLE
OCP4 Pipeline

### PR DESCRIPTION
**Pull request by ashtonmeuser**

### CODE REVIEW

File: server/server.js [hunk 1]
```diff
@@ -35,9 +35,7 @@ const keycloak = require('./keycloak.js');
 const apiBaseUrl = '/api/v1';
 const app = express();
 
-if (process.env.NODE_ENV === 'local') {
-  app.use(cors());
-}
+app.use(cors());
```
Comment (ashtonmeuser) [inline server/server.js:38]: This should be reverted once FE and BE are on same OCP cluster.

File: openshift/patroni.bc.yml [hunk 2]
```diff
@@ -13,7 +13,7 @@ parameters:
   - name: GIT_URI
     description: Source for Postgres repository with Dockerfile
     required: true
-    value: https://github.com/BCDevOps/platform-services.git
+    value: https://github.com/ashtonmeuser/platform-services.git
```
Comment (ashtonmeuser) [inline openshift/patroni.bc.yml:16]: This is problematic.

File: server/server.js (hunk 1 above)
Comment (arranfw) [reply server/server.js:38]: Is this true now?

File: .github/workflows/promote.yml [hunk 3]
```diff
@@ -7,8 +7,8 @@ on:
       - prod
 
 env:
-  CLUSTER: https://console.pathfinder.gov.bc.ca:8443
-  AUTH_TOKEN: ${{ secrets.AUTH_TOKEN }}
+  CLUSTER: https://api.silver.devops.gov.bc.ca:6443
+  AUTH_TOKEN: ${{ secrets.AUTH_TOKEN_OCP4 }}
```
Comment (arranfw) [inline .github/workflows/promote.yml:11]: When OCP3 is gone are we reverting this change? Is there a ticket made?

**Review by vyasworks — APPROVED** [review]

File: Makefile [hunk 4]
```diff
@@ -102,9 +102,14 @@ tag-prod:
 
 # OpenShift Aliases
 
-server-prep: # TODO: Move role binding command to a server prep template
+add-role:
+	@oc policy add-role-to-user admin system:serviceaccount:$(TARGET_NAMESPACE):default -n $(TOOLS_NAMESPACE)
```
Comment (arranfw) [inline Makefile:106]: As a newcomer should I know what these commands are used for? Perhaps there's some room for documentation here?

**Review by arranfw — APPROVED** [review]

File: server/server.js (hunk 1 above)
Comment (ashtonmeuser) [reply server/server.js:38]: This is not yet true. GDX has one DNS record left to swap over. There's a ticket that captures this but I don't see it as super high priority.

File: Makefile (hunk 4 above)
Comment (ashtonmeuser) [reply Makefile:106]: Lots of room for documentation. We were hoping for these to be roughly self-explanatory but failed on that front. Many of these were added as we found gaps between our Makefile targets and what was required to fully stand up our infra.

**Pull request merged**